runtime/meta/ffi.cpp

```diff
@@ -304,14 +304,23 @@ extern "C" {
     if (!mpz_fits_ulong_p(size)) {
       throw std::invalid_argument("Size is too large");
     }
+    if (!mpz_fits_ulong_p(align)) {
+      throw std::invalid_argument("Alignment is too large");
+    }
 
     if (allocatedKItemPtrs.find(kitem) != allocatedKItemPtrs.end()) {
       return allocatedKItemPtrs[kitem];
```

Comment:
What would happen when calling this function with the same kitem, but different alignment requirements?:

Reply:
You would get a value that is not necessarily aligned by the amount specified; In practice we assume the user won't call the code that way, but if you want, I can add a check that the memory returned is indeed aligned by the amount the user requested the second time around (and have it be an error otherwise).

Reply:
I would prefer having an error.